common/reactive-grpc-common/src/main/java/com/salesforce/reactivegrpc/common/AbstractStreamObserverAndPublisher.java

```diff
@@ -219,15 +207,12 @@ void drainRegular(final Subscriber<? super T> subscriber) {
         }
     }
 
-    void drainFused(final Subscriber<? super T> subscriber) {
+    private void drainFused(final Subscriber<? super T> subscriber) {
         int missed = 1;
 
-        final Queue<T> q = queue;
-
-        for (;;) {
-
+        do {
```

Comment:
Let's keep it as in the original code. From my point of view it preserves readability

Reply:
Fixed